ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/EmailIntegrationForm.tsx

```diff
@@ -478,6 +480,22 @@ function EmailIntegrationForm({
                             }
                         />
                     </FormLabelGroup>
+                    <FormLabelGroup
+                        label="Hostname for SMTP Hello"
+                        fieldId="notifier.email.hostnameHeloEhlo"
+                        touched={touched}
+                        errors={errors}
```

Comment:
Could you add a `helperText` prop here like in the **Sender** field above to explain about leaving the field blank?

Something like:
```jsx
                        helperText={
                            <span className="pf-v5-u-font-size-sm">
                                If left blank, localhost will be used
                            </span>
                        }
```